smarts/core/controllers/trajectory_tracking_controller.py

```diff
@@ -25,8 +25,6 @@
 import numpy as np
 
 from scipy import signal
-from cvxopt import matrix, solvers
-from qpsolvers import solve_qp
```

Comment:
Remember to also remove this from `setup.py` and `requirements.txt`.

Reply:
There are other usages in `lane_following_controller.py` and `actuator_dynamic_controller.py`. @iman512003 Are you going to eliminate them in one PR?

Reply:
> Remember to also remove this from `setup.py` and `requirements.txt`.

Thanks, sure.

Reply:
> There are other usages in `lane_following_controller.py` and `actuator_dynamic_controller.py`. @iman512003 Are you going to eliminate them in one PR?

Oh, Thanks, sure. I removed them from other controller files as well.